main.go

```diff
@@ -95,6 +98,49 @@ func init() {
 	//+kubebuilder:scaffold:scheme
 }
 
+func SetupLogging(opts *zap.Options) {
+	cfg, err := config.GetConfig()
+	if err != nil {
+		log.Error(err, "Error getting config for APIServer")
+		os.Exit(1)
+	}
+
+	clnt, err := client.New(cfg, client.Options{})
+	if err != nil {
+		log.Error(err, "can't create client from config")
+		os.Exit(1)
+	}
+	oprNamespace, err := sdkk8sutil.GetOperatorNamespace()
+	if err != nil {
+		log.Error(err, "failed to get operator namespace")
+		os.Exit(1)
+	}
+
+	oprCfg, _ := configmaps.RetrieveConfigMap(oprNamespace, "activemq-artemis-operator-config", clnt)
+	if oprCfg != nil && len(oprCfg.Data) > 0 {
+		if level, ok := oprCfg.Data["log-level"]; ok {
```

Comment:
I think maybe this should be generic "args", not specific to logging config - just any append-args

Reply:
agreed. will change accordingly.

Reply:
generic "args" is a nice idea my only concern is related to the data structure of the configmap,  the current one has 1 level so all data keys should be an opertator arg but if we add another level using the `args` data key in the future we could resuse this config map for other settings

current configmap data structure with one level:
```
kind: ConfigMap
apiVersion: v1
metadata:
  name: activemq-artemis-operator-config
immutable: false
data:
  --zap-log-level: debug
  --zap-encoder: console
  --zap-time-encoding: iso8601
  --leader-elect:
```

alternative configmap data structure with 2 levels:
```
kind: ConfigMap
apiVersion: v1
metadata:
  name: activemq-artemis-operator-config
immutable: false
data:
  args: |
    --zap-log-level=debug
    --zap-encoder=console
    --zap-time-encoding=iso8601
    --leader-elect
```

Reply:
I think for now one-level is ok, and it's simple.

Reply:
@brusdev Gary mentioned that we may consider using this configmap for that purpose instead of creating a new configmap. https://github.com/artemiscloud/activemq-artemis-operator/blob/main/deploy/install/100_operator_config.yaml , this is generated from kustomize and I think it may be possible to add a 'args' entry into it.

Reply:
but the problem is that how user can pre-create the configmap when the operatorHub will install it again.